doc/changelog/08-tools/12410-add-fixes.rst

```diff
@@ -0,0 +1,4 @@
+- **Changed:**
+  ``dev/tools/make-changelog.sh`` now asks for a list of bugs fixed by the PR
+  (`#12410 <https://github.com/coq/coq/pull/12410>`_, fixes `#12386
+  <https://github.com/coq/coq/issues/12386>`_, by Jason Gross).
```

Comment:
Thanks for the example :) but we usually do not introduce changelog entries for purely internal PRs.